File: atividade.cpp
#include <stdlib.h>
#include <iostream>
int main(){
	try{
        system("touch atividade.txt");
        system("echo \"Só alegria hahaha\" > alegria.txt");
	}catch(...){
	std::cout << "\nErro durante execução do programa!\n";
}
}

File: atividade.py
import os
def main():
	try:
		os.system("touch ricardo.txt")
		os.system("echo \'Só alegria hahaha\' >> ricardo.txt ")
		print("***********************************\n* Programa executado com sucesso! *")
		print("***********************************")
	except:
		print("Houve unm erro durante a execução do programa!")
main()
